repo/migrations/migration_helpers.go

```diff
@@ -0,0 +1,56 @@
+package migrations
+
+import (
+	"database/sql"
+	"os"
+	"path"
+	"strconv"
+)
+
+func NewDB(repoPath string, dbPassword string, testnet bool) (*sql.DB, error) {
```

Comment:
Maybe more like `OpenDB`? Was surprised by the name and that it lived in `migrations` until I came to look.

Reply:
One thing to be concerned with the idea of helpers in our migrations will be the problem that arises when the helper breaks across schema versions (if we were to move the location of the database file, for example). I tend to inline the helpers needed for each migration and write new ones. Something like:

`repo/migrations/migration_009.go`
```
func migration009_openDB (...) (...) {
   // actual open logic here
}
```

`repo/migrations/migration_010.go`
```
func migration010_openDB (...) (...) {
   return migration009_openDB(...)
}

// or explicitly show how this schema is changing 
// by manipulating the inputs as needed for migration009_openDB
func migration010_openDB (path string, ...) (...) {
   migration010_dbPath := migration010_dbPathChanges(path)
   return migration009_openDB(migration010_dbPath, ...)
}

```